src/auto_import.js

```diff
@@ -109,34 +106,32 @@ function importFromAPI(urlForAPI) {
       // Get default language
       languageSettings = AUTO_IMPORT_LANGUAGE_SETTINGS_FOR_IMPORT["English"];
     }
-    errorCodeNotEncountered = true;
     // Clear status
     for (var i = 0; i < WISH_TALLY_NAME_OF_WISH_HISTORY.length; i++) {
       bannerName = WISH_TALLY_NAME_OF_WISH_HISTORY[i];
       bannerSettings = AUTO_IMPORT_BANNER_SETTINGS_FOR_IMPORT[bannerName];
       bannerSettings.set_range_status("", settingsSheet);
     }
     for (var i = 0; i < WISH_TALLY_NAME_OF_WISH_HISTORY.length; i++) {
-      if (!errorCodeNotEncountered) {
-        bannerSettings.set_range_status(
-          "Stopped Due to Error:\n" + bannerSettings.range_status(settingsSheet),
-          settingsSheet);
-        break;
-      }
-
       bannerName = WISH_TALLY_NAME_OF_WISH_HISTORY[i];
       bannerSettings = AUTO_IMPORT_BANNER_SETTINGS_FOR_IMPORT[bannerName];
       if (bannerSettings.is_toggled(settingsSheet)) {
         bannerSheet = SpreadsheetApp.getActive().getSheetByName(bannerName);
         if (bannerSheet) {
-          checkPages(
+          var encounteredErrorCode = checkPages(
```

Comment:
Ususally, functions report `true` in the success case and not vice versa, but with the variable name it's clean enough.

This is definitely a very valuable refactoring though. Unnecessarily sing globals for error tracking is very spaghetti and prone to errors.